src/main/java/com/maxmind/ws/CreditCardFraudDetection.java

```diff
@@ -76,4 +83,31 @@ protected String filter_field(String key, String value) {
         }
         return value;
     }
+
+    public static class Builder {
+        private boolean systemProxies = false;
+        private String proxyHost = null;
+        private int proxyPort = -1;
+        private boolean isSecure = true;
+
+        public Builder() { }
+
+        public Builder useSystemProxies() {
+            systemProxies = true;
+            return this;
+        }
+        public Builder useProxy(String proxyHost, int proxyPort) {
+            this.proxyHost = proxyHost;
+            this.proxyPort = proxyPort;
+            return this;
+        }
+        public CreditCardFraudDetection build() {
+            return new CreditCardFraudDetection(this);
+        }
+
+        public CreditCardFraudDetection buildUnsecure() {
+            this.isSecure = false;
```

Comment:
I think we should get rid of the separate `buildUnsecure()` method and just make a `disableSecure()` method (or something similar) on the builder.